components/Carousel/Carousel.jsx

```diff
@@ -91,13 +91,17 @@ class Carousel extends Component {
       isFresh: false,
     });
 
+
     setTimeout(() => {
       this.setState({
-        bgSlideIndex: i,
+        bgSlideIndex: calcNext(this.props.slides.length, i), // assume they'll be going forward, since that's more likely
```

Comment:
This one stumps me. Logically, it should work. But this actually ended up breaking the animations in rare cases. I couldn't see a pattern to it. Reverted in a later commit.